### ISSUE TITLE
Test reliability chores

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4417]
## accessibility.liveRegion.activityStatus.sendFailed.live region / should narrate "failed to send message" through live region

```
Error: expect(received).toEqual(expected) // deep equality

- Expected  - 1
+ Received  + 0

  Array [
    "You said:
  Hello, World!",
    "Bot said:
  Aloha!",
-   "Failed to send message.",
  ]
    at run.ignoreErrors (http://webchat2/__tests__/html/accessibility.liveRegion.activityStatus.sendFailed:46:27)
```
